Resources/Computing/BatchSystems/LSF.py

```diff
@@ -9,12 +9,13 @@
     LocalComputingElement and SSHComputingElement classes
 """
 
-__RCSID__ = "$Id$"
-
+from __future__ import print_function
```

Comment:
Do we need at all these imports? There's no call to `print` here anyway. But maybe I miss the details of how this is used.

Reply:
You are right, there is no `print` in this code but these imports were in `Resources/Computing/BatchSystems/executeBatch.py` previously and this is what caused the issue because after merging we had them at the middle of the `execute_batch` script: 

Should I develop something more complex to extract the `import __future__` from the `execute_batch` script to put it at the top of the file?
Should I replace or remove the `print` from `executeBatch.py`?
Should I remove the `import __future__` from `executeBatch.py`?